app/services/witter.js

```diff
@@ -0,0 +1,14 @@
+const axios = require('axios').default;
+const logger = require('../logger');
+
+const WitterInstance = axios.create({
+  baseURL: 'http://numbersapi.com'
```

Comment:
This should be and env variable and it should be mapped in the `config/index.js` file